guides/release-process.md

```diff
@@ -87,15 +87,22 @@ In the following instructions, "X.Y.Z" is used to denote the [next version of Cy
     - To find the link to the package file `cypress.tgz`:
         1. In GitHub, go to the latest commit (the one whose sha you used in the last step).
             ![commit-link](https://user-images.githubusercontent.com/1157043/80608728-33fe6100-8a05-11ea-8b53-375303757b67.png)
-        2. Scroll down past the changes to the comments. The first comment should be a `cypress-bot` comment that includes a line beginning `npm install ...`. Grab the `https://cdn.../npm/X.Y.Z/<platform>/<long sha>/cypress.tgz` link.
+        2. Scroll down past the changes to the comments. Find a comment from `cypress-bot` with an `npm install` command for `linux-x64`. Copy the `https://cdn.../npm/X.Y.Z/linux-x64/<long sha>/cypress.tgz` URL.
             ![commit-bot-comment](../assets/cypress-bot-pre-release-comment.png)
-    - Make sure the `linux-x64` binary and npm package are present at the commented locations. See [Before Publishing a New Version](#before-publishing-a-new-version).
-    - Publish the `linux-x64` distribution to the npm registry straight from the URL:
+    - Use the `create-stable-npm-package` script to create a stable NPM package from the pre-prod `cypress.tgz` URL:
```

Comment:
Would it be possible to update move-binaries to handle this for us so there is one less manual step & potential release error?

Reply:
I think so. I was trying to be as deliberate as possible with making this its own step, so whoever is releasing is aware of what they're doing, but I guess it could be a source for errors. I'll look at this.

Reply:
[ff54121](https://github.com/cypress-io/cypress/pull/20296/commits/ff5412156c17b56f16fb1198353edcf2a2cacd9d)